graylog2-web-interface/src/components/streams/StreamDetails/routing-destination/DestinationIndexSetSection.tsx

```diff
@@ -104,12 +104,9 @@ const DestinationIndexSetSection = ({ stream }: Props) => {
         />
       }>
       <Alert bsStyle="default">
-        Messages routed to the <b>Search Cluster</b> will be searchable in Graylog and count towards Graylog License
-        usage.
-        <br />
-        These messages will be stored in the defined Index Set until the retention policy criteria is met.
-        <br />
-        Note: Messages not routed to the <b>Search Cluster</b> will not be searchable in Graylog.
+        Messages routed to the <b>Search Cluster</b> will be searchable and count towards the License usage. These
```

Comment:
I think we have to keep the product name here to make it clear which license we are talking about